packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -60,8 +60,7 @@ function ButtonBlockAppender(
 
 				return (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
| Before | After     |
| :-------- | :------- |
|  ![button-block-appender-before](https://github.com/user-attachments/assets/4dc8c584-1f4e-405f-b397-3a6f7b283e4a) | ![button-block-appender-after](https://github.com/user-attachments/assets/3395d64e-4737-40c3-bc21-89b292dae378) |

Reply:
The `+` icon looks too small. `flex-shrink: 0` should fix but there might be a better way?

Related, the empty placeholder that appears adjacent in a Row/Stack has a `46px` min-height which causes a mis-match:

<img width="667" alt="Screenshot 2024-09-12 at 16 08 43" src="https://github.com/user-attachments/assets/f0008151-3a30-46bd-8e91-a8810f56d53a">

It might be good to update that here.

Reply:
Hi @jameskoster, I have removed the styles that were overriding the component button padding to fix the small icons. Also fixed the empty placeholder height.

<img width="784" alt="image" src="https://github.com/user-attachments/assets/6943b5b1-1f8e-4c78-8ea5-0fa86d6b966e">